training/provision/playbook.yml

```diff
@@ -15,16 +15,41 @@
   - name: Gather facts for first time
     ansible.builtin.setup:
 
+  - name: Get size of root filesystem
+    ansible.builtin.shell:
+      cmd: |
+        df -h /
+        df -h /var/tmp/
+        podman system info
+
   - name: Check Podman Present
     ansible.builtin.package:
       name: podman
       state: present
 
+  - name: Login to default registry and create ${XDG_RUNTIME_DIR}/containers/auth.json
+    containers.podman.podman_login:
+      username: "{{ registry_user }}"
+      password: "{{ registry_password }}"
+      registry: quay.io
+
+  - name: Building an image with ssh key
+    containers.podman.podman_container:
+      name: localhost/temp_image
+      image: "quay.io/ai-lab/{{ image_name }}:latest"
+      command:
+      - mkdir /usr/etc-system
+      - test -n "{{ ssh_public_key }}"
+      - echo 'AuthorizedKeysFile /usr/etc-system/%u.keys' >> /etc/ssh/sshd_config.d/30-auth-system.conf
+      - "echo {{ ssh_public_key }} > /usr/etc-system/fedora.keys && chmod 0600 /usr/etc-system/fedora.keys"
+      state: present
+      recreate: true
+
   - name: Bootc install
```

Comment:
Per discussion in chat this will not work with any centos/rhel derived container images targeting a Fedora cloud instance which uses btrfs.  See https://github.com/containers/bootc/issues/192